cbv3_django_prototype/cbv3_django_prototype/userauth/serializers.py

```diff
@@ -10,8 +11,8 @@ class UserSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = get_user_model()
-        fields = ('username', 'is_superuser',)
-
+        fields = ('id', 'username', 'first_name', 'last_name', 'email', 'is_superuser',)
```

Comment:
We don't need to return email here if we don't want to.  Happy to remove it. 

 Because things like this are so deeply nested in Django/DRF config, I think it would be great if we did a little thinking about where and when (if ever) we expose this info.  It might even be a good idea to think through how we do it for the **website client** vs. direct **api** calls - we may want the website (user profile, pw change, hangouts scheduling) to get email -- but not a bot requesting resources for something else.

And once we have that, we can define the 2-3 places that it gets referred to or returned, and lock things down appropriately.   Since the user info is in a `users` app, and a `users` table, we may want a `users` endpoint that requires additional authing, and force a request there for email.  

For now I will remove this info from the endpoint, and then we can open issues/discussion around emails and user info in a separate issue.

Reply:
The big takeaway for us here is that because of the way `ModelSerializer` works, it has access to **anything** the model refers to - and since `user` is a PK into `users`, all that info is available via the FK.  So we have to remind developers like me that just because it is **easy** doesn't mean we should do it.

Reply:
We may also want to consider a different, custom serializer for restricted user access vis full user access.

Reply:
I'm happy with removing this for now and exploring the need to add it back in later as required.